javascript/app.js

```diff
@@ -52,7 +65,7 @@ FinancialFreedom.controller('HeaderController', ['$scope', '$location',  functio
     };
 
     $scope.tabsAreVisible = function() {
-        if ($location.path() == '/about' || $location.path() == '/time-to-retirement') {
+        if ($location.path() == '/about' || $location.path() == '/time-to-retirement' || $location.path() == '/') {

```

Comment:
if we expect this function to expand to more pages, we could change how this expression is checked by doing something like the following:
`
visible_pages = [
  '/about',
  '/time-to-retirement',
  '/'
]

if ( visible_pages.indexOf($location.path() ) != -1)
`

it's mostly fine as is, this way would just make it a bit shorter and easier to understand and see what pages it's actually checking for